src/Entities/Panels/AbstractPanel.cs

```diff
@@ -158,7 +158,7 @@ public AbstractPanel(Vector2 position, float size, Directions orientation, bool
 
         private static void LogHooked(string fullname) {
             if (!hooked.Contains(fullname)) {
-                Logger.Log(LogLevel.Info, "CommunalHelper", $"Hooking {fullname} to override when AbstractPanel present.");
+                Logger.Log(LogLevel.Verbose, "CommunalHelper", $"Hooking {fullname} to override when AbstractPanel present.");
```

Comment:
I think I'd prefer for this to be at least `Debug`, since it's something that I worry about causing weird conflicts for other mods

Reply:
Yeah that makes sense, that's what we changed detour logs to anyway. Updated.